packages/redirects-utils/src/index.js

```diff
@@ -0,0 +1,84 @@
+const fs = require('fs')
+const path = require('path')
+const { promisify } = require('util')
+
+const writeFileAsync = promisify(fs.writeFile)
+
+const resolveConfig = require('@netlify/config')
+const { getConfigPath } = require('@netlify/config')
+const { parseRedirectsFormat, parseNetlifyConfig } = require('netlify-redirect-parser')
+const configorama = require('configorama')
+
+async function parseFile(parser, name, filePath) {
+  const result = await parser(filePath)
+  if (result.errors.length) {
+    console.error(`Warnings while parsing ${name} file:`)
+    result.errors.forEach(err => console.error(`  ${err.lineNum}: ${err.line} -- ${err.reason}`))
+  }
+  return result.success
+}
+
+async function getAll(projectPath) {
+  const rulesUnique = new Set()
+
+  const configPath = await getConfigPath(undefined, projectPath)
+  const redirectsFilePath = path.resolve(projectPath, '_redirects')
+
+  if (fs.existsSync(redirectsFilePath)) {
+    const fileName = redirectsFilePath
+      .split(path.sep)
+      .pop()(await parseFile(parseRedirectsFormat, fileName, redirectsFilePath))
+      .forEach(r => rulesUnique.add(JSON.stringify(r)))
+  }
+  if (fs.existsSync(configPath)) {
+    const fileName = configPath
+      .split(path.sep)
+      .pop()(await parseFile(parseNetlifyConfig, fileName, configPath))
```

Comment:
I am a little confused by this line: the `pop()` function should return a string, but that string is treated as a function (with `await parseFile()` as argument)?

Reply:
Ah thanks for catching this. Seems like linter messed things up because there was no semicolon after `pop()`.